test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -32,7 +34,7 @@ DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& i
         filter_names.reserve(registered_names.size());
         std::for_each(registered_names.begin(), registered_names.end(),
                       [&](const absl::string_view& filter) {
-                        if (exclusion_list.count(filter) == 0) {
+                        if (exclusion_list.contains(filter)) {
```

Comment:
should be !contains?

Reply:
Meehhh, just one simple job to do and failing that.
Sorry, fixed.